pkgs/_macro_client/lib/macro_client.dart

```diff
@@ -96,25 +116,12 @@ class RemoteMacroHost implements Host {
 
   @override
   Future<Model> query(Query query) async {
+    final id = nextRequestId;
```

Comment:
Why grab the id early here?

Reply:
Not needed any more, indeed, thanks; done.